packages/app-mobile/components/screens/Note/Note.tsx

```diff
@@ -1603,7 +1603,7 @@ const NoteScreenWrapper = (props: Props) => {
 
 	const dialogs = useContext(DialogContext);
 	return (
-		<NoteScreenComponent key={noteId} dialogs={dialogs} {...props} />
+		<NoteScreenComponent key={noteId} dialogs={dialogs} {...props} noteId={noteId} />
```

Comment:
If `state.selectedNoteIds` is always non-empty when rendering a note, overriding the `noteId` prop shouldn't be necessary.

I'm marking this pull request as a draft until this has been changed.